README.md

```diff
@@ -1,5 +1,19 @@
 # ffd-info-exchange
 
+## About this project
+
+Welcome to the ffd-info-exchange repository! The Federal Front Door: Information Exchange project is a twelve-week discovery engagement meant to uncover possibilities for information sharing between federal agencies as a way of providing better public access to government services. 
```

Comment:
I'm also wondering if it would be worth creating a roadmap (even a skeletal one) — thoughts? Would that be too much work, given the nature of this project? (I'm happy to help draft one if folks think it would be worthwhile)